types/merge-refs/index.d.ts

```diff
@@ -0,0 +1,9 @@
+// Type definitions for merge-refs 1.0
+// Project: https://github.com/wojtekmaj/merge-refs
+// Definitions by: Jakub Skoneczny <https://github.com/Skona27>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+import * as React from 'react';
+
+declare function merge_refs<T>(...refs: Array<React.Ref<T>>): (instance: T) => void;
+export = merge_refs;
```

Comment:
This needs to be exported using `default` instead.
https://unpkg.com/browse/merge-refs@1.0.0/dist/umd/index.js

Reply:
Changed this to be `export default`